core/integration/client_tests.go

```diff
@@ -148,14 +149,11 @@ func TestBatchUpdate(ctx context.Context, env *Env, t *testing.T) []testdata.Res
 				})
 			}
 
-			cctx, cancel := context.WithTimeout(ctx, env.Timeout)
-			defer cancel()
-
-			mutations, err := env.Client.BatchCreateMutation(cctx, users)
+			mutations, err := env.Client.BatchCreateMutation(ctx, users)
 			if err != nil {
 				t.Fatalf("BatchCreateMutation(): %v", err)
 			}
-			if err := env.Client.BatchQueueUserUpdate(cctx, mutations, signers1); err != nil {
+			if err := env.Client.BatchQueueUserUpdate(ctx, mutations, signers1); err != nil {
```

Comment:
The timeout has gone, do you not need it?

Reply:
The timeout is not needed because tests as a whole have timeouts.